### PR TITLE
Added Animations on HomePage and Enterprise

### PR DESCRIPTION
**Pull request by Espina2**

### CODE REVIEW

**Comment by Espina2** [comment]
@nextcloud/designers @LukasReschke can review and merge?

**Comment by jancborchardt** [comment]
@LukasReschke can you deploy it to staging?

File: assets/css/custom.css [hunk 1]
```diff
@@ -407,8 +407,20 @@ section#why-nextcloud {
       padding-right: 30px; }
     .Header-Navigation .navbar li a {
       color: white; }
+      .Header-Navigation .navbar li a:after {
```
Comment (skjnldsv) [inline assets/css/custom.css]: This is breaking the download button! :/
screen

Comment (Espina2) [inline assets/css/custom.css]: This is added in the last commit ( I shouldn't commit that yet). I trying to figure out why a not :last-child doesn't work for an after pseudo-element.

Comment (Espina2) [inline assets/css/custom.css]: But are editing directly the .css file, I'm talking about the .scss file. But if made it working just made a commit with the last-child not having an after element. This should work

Comment (skjnldsv) [inline assets/css/custom.css]: Okay! Good luck. The rest looks good :)

**Comment by LukasReschke** [comment]
Resolved the conflict and deployed at https://nextcloud.com/enterprise/?staging=true and https://nextcloud.com/?staging=true

Let me see how to fix the download button…

File: assets/css/enterprise.scss [hunk 2]
```diff
@@ -40,20 +40,23 @@
...
-        height: 60px;
-        padding-top: 16px
+    
+	  .topheader {
+      	padding: 0;
+      
+		a{
+			width: 290px;
+			height: 60px;
+			padding-top: 16px
```
Comment (LukasReschke) [inline assets/css/enterprise.scss]: This looks wrongly intended.

File: assets/css/enterprise.scss [hunk 3]
```diff
@@ -40,20 +40,23 @@
...
       }
     }
   }
 
   .container-fluid{
-    img {
-      height: 100%;
-      width: 100%;
+    
+	  img {
```
Comment (LukasReschke) [inline assets/css/enterprise.scss]: As well. Might want to look into caps vs. spaces in your text editing tool :)

**Comment by Espina2** [comment]
I think you not include the main.js function in the deploy. Can you check @LukasReschke

**Comment by Espina2** [comment]
@LukasReschke @jospoortvliet We need to deploy the main.js for testing.

**Comment by LukasReschke** [comment]
> @LukasReschke @jospoortvliet We need to deploy the main.js for testing.

It is. But it doesn't seem to be included anywhere or am I missing something? :)

**Comment by Espina2** [comment]
You have misses something. Maybe some script code in the homepage html. @LukasReschke can you take a look?

**Comment by LukasReschke** [comment]
> You have misses something. Maybe some script code in the homepage html. @LukasReschke can you take a look?

Sure. So "main.js" should be included on every page or just on a specific one?

**Comment by Espina2** [comment]
Yup, for now the main.js should be in everypage. More later we need to think about require.js.

**Comment by jospoortvliet** [comment]
2 questions/comments.
1. can we do this in css, not javascript and
2. I note that the texts under the three items just under the header on the front page has disappeared.

**Pull request merged**